packages/@react-spectrum/color/test/ColorField.test.js

```diff
@@ -359,11 +375,67 @@ describe('ColorField', function () {
     fireEvent.keyUp(colorField, {key});
     expect(onChangeSpy).toHaveBeenCalledWith(minColor);
     expect(onChangeSpy).toHaveBeenCalledTimes(1);
-    expect(colorField.value).toBe(minColor.toString('hex'));
+    expect(colorField.value).toBe(minColor.toString(props.allowsAlpha ? 'hexa' : 'hex'));
 
     // repeat action to make sure onChange is not called when already at min
     fireEvent.keyDown(colorField, {key});
     fireEvent.keyUp(colorField, {key});
     expect(onChangeSpy).toHaveBeenCalledTimes(1);
   });
+
+  it('should switch between alpha display and not', () => {
+    let props = {defaultValue: '#aabbcc'};
+    let tree = renderComponent(props);
+    let colorField = tree.getByLabelText('Primary Color');
+    expect(colorField.value).toBe('#AABBCC');
+    tree.rerender(
+      <Provider theme={theme}>
+        <ColorField
+          {...props}
+          label="Primary Color"
+          allowsAlpha />
+      </Provider>
+    );
+    expect(colorField.value).toBe('#AABBCCFF');
+    tree.rerender(
+      <Provider theme={theme}>
+        <ColorField
+          {...props}
+          label="Primary Color" />
+      </Provider>
+    );
+    expect(colorField.value).toBe('#AABBCC');
+  });
+
+  it('should match data to display when switching between alpha display and not', () => {
```

Comment:
This test looks like the above test with user input added, but the description tells me it's the same test.

What about a test that switches from hex to hex with alpha?